src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -325,11 +341,122 @@ public int getApiVersion()
         }
     }
 
+    public KojiSessionInfo krbLogin()
```

Comment:
I think we should spin off most of this implementation into a KrbManager or KrbAuthenticator component. That component could have two methods:

```
KrbLoginRequest prepareRequest() throws KojiClientException;
KojiSession hanldleResponse( KrbLoginResponse ) throws KojiClientException;
```

This would help streamline this method, and hide the details of Kerberos auth from the rest of the KojiClient, which is meant to be the main user interface for kojiji. I think leaving this in here will reduce readability of the code.